tasks/post_to_queue_and_trigger_step_function/post_to_queue_and_trigger_step_function.py

```diff
@@ -52,31 +52,32 @@ def process_record(
     Central method for translating record to pass along and triggering step function.
 
     Args:
-        record: The record to post.
+        record: The record to post. See schemas/input.json under Records/items for details.
         target_queue_url: The url of the queue to post the records to.
         step_function_arn: The arn of the step function to trigger.
     """
-    new_body = translate_record_body(record["body"])
-    sqs_library.post_to_fifo_queue(target_queue_url, new_body)
-    trigger_step_function(step_function_arn)
+    body = json.loads(record["body"])
+    _BODY_VALIDATE(body)
+
+    for s3_record in body["Records"]:  # Unsure if AWS sends multiple records, but data structure supports it.
+        new_body = translate_record_body(s3_record)
+        sqs_library.post_to_fifo_queue(target_queue_url, new_body)
```

Comment:
post_to_fifo_queue() name in sqs_library.py should be changed to something like post_to_standard_queue() or similar since it is not a fifo queue anymore.

Reply:
post_to_queue_and_trigger_step_function reads from the s3_inventory_queue (which is no longer fifo) and posts to the internal_report_queue (which is still fifo).

Reply:
oh I see.